pkg/cephmgr/rgw/leader.go

```diff
@@ -104,6 +105,25 @@ func RemoveObjectStore(context *clusterd.Context) error {
 	return nil
 }
 
+func GetRGWEndpoints(etcdClient etcd.KeysAPI, clusterInventory *inventory.Config) (host, ipEndpoint string, found bool, err error) {
```

Comment:
As a form of documentation.  When i was reading about named return args in golang today, that does appear to be an idiomatic usage of them.